app/app.rb

```diff
@@ -0,0 +1,77 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require './app/support/data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+
+helpers do
+  def current_user
+    @current_user || User.get(session[:user_id])
+  end
+
+  def peeps
+    if Peep.all.length > 0
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)<br>Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)